### ISSUE TITLE
Lock error in logs?

### ISSUE THREAD

**Issue by jscherry** [JoshuaMulliken/ha-wyzeapi#311]
**Describe the bug**

Not sure what this is? I've been getting this error with both of my locks with all recent versions of the integration?

**System configuration**
## System Health

version | core-2021.12.3
-- | --
installation_type | Home Assistant OS
dev | false
hassio | true
docker | true
user | root
virtualenv | false
python_version | 3.9.7
os_name | Linux
os_version | 5.10.83
arch | aarch64
timezone | America/Chicago

<details><summary>Home Assistant Community Store</summary>

For additional information see the readme: https://github.com/JoshuaMulliken/ha-wyzeapi#reporting-an-issue
-->

```
<PUT YOUR LOG HERE>
```
Logger: homeassistant.util.logging
Source: util/logging.py:105
First occurred: 9:34:38 PM (12 occurrences)
Last logged: 9:36:47 PM

Exception in handle_lock_update when dispatching 'wyzeapi.lock_updated-YD.LO1.6a4739411019da2d1caea8a133a23758': (<Device: DeviceTypes.LOCK, YD.LO1.6a4739411019da2d1caea8a133a23758>,) Traceback (most recent call last): File "/config/custom_components/wyzeapi/sensor.py", line 86, in handle_lock_update if self.enabled is False | self.enabled is None: TypeError: unsupported operand type(s) for |: 'bool' and 'NoneType'
Exception in handle_lock_update when dispatching 'wyzeapi.lock_updated-YD.LO1.23e3cde4a379528d7d2f310b221dd4a6': (<Device: DeviceTypes.LOCK, YD.LO1.23e3cde4a379528d7d2f310b221dd4a6>,) Traceback (most recent call last): File "/config/custom_components/wyzeapi/sensor.py", line 86, in handle_lock_update if self.enabled is False | self.enabled is None: TypeError: unsupported operand type(s) for |: 'bool' and 'NoneType'

**Comment by jscherry** [comment]
Logger: homeassistant.util.logging
Source: util/logging.py:105
First occurred: 9:34:38 PM (58 occurrences)
Last logged: 9:43:56 PM

Exception in handle_lock_update when dispatching 'wyzeapi.lock_updated-YD.LO1.6a4739411019da2d1caea8a133a23758': (<Device: DeviceTypes.LOCK, YD.LO1.6a4739411019da2d1caea8a133a23758>,) Traceback (most recent call last): File "/config/custom_components/wyzeapi/sensor.py", line 86, in handle_lock_update if self.enabled is False | self.enabled is None: TypeError: unsupported operand type(s) for |: 'bool' and 'NoneType'
Exception in handle_lock_update when dispatching 'wyzeapi.lock_updated-YD.LO1.23e3cde4a379528d7d2f310b221dd4a6': (<Device: DeviceTypes.LOCK, YD.LO1.23e3cde4a379528d7d2f310b221dd4a6>,) Traceback (most recent call last): File "/config/custom_components/wyzeapi/sensor.py", line 86, in handle_lock_update if self.enabled is False | self.enabled is None: TypeError: unsupported operand type(s) for |: 'bool' and 'NoneType'

**Comment by yoinx** [comment]
That's me messing up/bastardizing languages.

That pipe should be an "or". Double pipes are "or" in java. Everything from the pipe on in that line is actually redundant since None should actually evaluate to false anyway.

**Comment by jscherry** [comment]
I edited the file and that fixed the error. Thanks

**Comment by JoshuaMulliken** [comment]
We still need to fix this in the main dist. @jscherry can you push your changes as a pr?

**Comment by jscherry** [comment]
Josh I’m sorry I don’t know how to push the commit. I’d be glad to if you tell me how.

**Comment by jscherry** [comment]
Josh I think I sent a pr for the change? A lot of this is new to me.

**Comment by JoshuaMulliken** [comment]
@jscherry I don't see one but here is a great tutorial on how to get started submitting pull requests! https://www.digitalocean.com/community/tutorials/how-to-create-a-pull-request-on-github

**Comment by ninadpchaudhari** [comment]
> We still need to fix this in the main dist. @jscherry can you push your changes as a pr?

Thanks, since the original author confirmed the change should not affect anything else and the later part of the code was redundant, PR #334  Created.

**Comment by jscherry** [comment]
The change I made only corrected the lock log errors but then caused other errors?

**Comment by ninadpchaudhari** [comment]
> The change I made only corrected the lock log errors but then caused other errors?
TLDR : The error was not causing any effects on functionality. It was handling an edge case, which is just a safeguard. 
The error is a `TypeError` and is caused by a error in syntax. Fixing the syntax does the job. 

Moreover, this will not affect anything since all that `If` statement was doing was to set `enabled` property to True 
now the same is already done on the line before that `if` statement when Lock's battery exists. In the case your Lock's battery was not detected, but Keypad's battery was detected, then this statement would have been useful.
